cypress/testData/testData.js

```diff
@@ -1,9 +1,31 @@
 const faker = require('faker');
-export const BASE_URL = 'https://demoqa.com/text-box';
+
```

Comment:
To my mind, there are unnecessary empty lines here.

Reply:
Done